packages/pwa-buildpack/lib/WebpackTools/PWADevServer.js

```diff
@@ -215,6 +221,18 @@ const PWADevServer = {
             };
         }
 
+        /*
+         * Make sure that Dev server is binding to IP, since passed domain name not always could be resolved
+         * Configure public option in order to allow configured domain:port to work properly on 0.0.0.0
+         */
+        Object.assign(webpackDevServerOptions, {
+            public: [
+                webpackDevServerOptions.host,
+                getDecoratedListeningPort(webpackDevServerOptions.port)
+            ].join(':'),
+            host: '0.0.0.0'
+        });
```

Comment:
You are right that the dev server should just bind to `0.0.0.0` instead of to the custom hostname. I don't quite understand the need for additional port configuration, though. Here's what I would expect in a containerized setup.

Let's say that your nginx-proxy is serving to the browser from host `https://pwa.docker.nginx`, and proxying from a dev server at host `0.0.0.0:3456`.

```sh
CUSTOM_ORIGIN_ENABLED=
DEV_SERVER_HOST=0.0.0.0
DEV_SERVER_PORT=3456
DEV_SERVER_PUBLIC_PATH=https://pwa.docker.nginx
```

I don't think that any more port mapping than this is necessary. Am I missing something?